llvm/lib/Target/AMDGPU/VINTERPInstructions.td

```diff
@@ -173,6 +173,12 @@ defm : VInterpF16Pat<int_amdgcn_interp_inreg_p10_f16,
 defm : VInterpF16Pat<int_amdgcn_interp_inreg_p2_f16,
                      V_INTERP_P2_F16_F32_inreg, f16,
                      [VINTERPModsHi, VINTERPMods, VINTERPMods]>;
+defm : VInterpF16Pat<int_amdgcn_interp_inreg_p10_rtz_f16,
+                     V_INTERP_P10_RTZ_F16_F32_inreg, f32,
+                     [VINTERPModsHi, VINTERPMods, VINTERPModsHi]>;
+defm : VInterpF16Pat<int_amdgcn_interp_inreg_p2_rtz_f16,
+                     V_INTERP_P2_RTZ_F16_F32_inreg, f16,
```

Comment:
The inreg part seems to be a compiler invented suffix, so if we really need to expose that in the intrinsic name, it should move to the end

Reply:
Why is the inreg here at all?

Reply:
It was used to disambiguate hardware instructions that changed behaviour without changing name. Pre-GFX11 they loaded input data from LDS. Since GFX11 they don't load from LDS, they just do to the interpolation calculation "in registers". In fact there was only one instruction V_INTERP_P2_F32 that had the identical name before/after this transition, but we used the inreg suffix on all of them for consistency.

Reply:
I think it makes sense to keep aligned with the non-rtz version?